packages/grid/_modules_/grid/components/GridNoResultsOverlay.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { GridApiContext } from './GridApiContext';
+import { GridOverlay, GridOverlayProps } from './containers/GridOverlay';
+
+export const GridNoResultsOverlay = React.forwardRef<HTMLDivElement, GridOverlayProps>(
+  function GridNoResultsOverlay(props, ref) {
+    const apiRef = React.useContext(GridApiContext);
+    const allRowsFilteredLabel = apiRef!.current.getLocaleText('allRowsFilteredLabel');
```

Comment:
can we rename this key to match the component name please?
noResultOverlayLabel